code/modules/power/tesla/energy_ball.dm

```diff
@@ -82,6 +81,14 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 			var/orbitsize = (I.Width()+I.Height())*pick(0.5,0.6,0.7)
 			orbitsize -= (orbitsize/world.icon_size)*(world.icon_size*0.25)
 			EB.orbit(src,orbitsize, pick(FALSE,TRUE), rand(10,25), pick(3,4,5,6,36))
+	else if (energy < energy_to_lower && orbiting_balls.len)
+		energy_to_raise = energy_to_raise * 0.5
+		energy_to_lower = (energy_to_raise * 0.5) - 10
+		var/Orchiectomy_target = pick(orbiting_balls)
+		orbiting_balls.Remove(Orchiectomy_target)
+		qdel(Orchiectomy_target)
+	else	
+		energy -= 5*orbiting_balls.len
```

Comment:
it even DRAINS SUPER FAST IF YOU HAVE MORE BALLS WHAT THE FUCK

YOU LITERALLY GUTTED THE FUN UNIQUE PARTS OF THE ENGINE